src/Tribe/REST/V1/Main.php

```diff
@@ -24,9 +24,16 @@ class Tribe__Events__REST__V1__Main extends Tribe__REST__Main {
 	 * Binds the implementations needed to support the REST API.
 	 */
 	public function bind_implementations() {
+		$messages = new Tribe__Events__REST__V1__Messages();
+
+		tribe_singleton( 'Tribe__REST__Messages_Interface', $messages );
```

Comment:
my misuse of how we'd like to use the service locator pattern in favour of how I usually use the DI container to bind implementations to interfaces, removed as out of how we actually write the code